app/models/early_career_payments/eligibility.rb

```diff
@@ -0,0 +1,32 @@
+module EarlyCareerPayments
+  class Eligibility < ApplicationRecord
+    EDITABLE_ATTRIBUTES = [
+      :nqt_in_academic_year_after_itt
+    ].freeze
```

Comment:
Should we prefer lines around multiline blocks? Seems inconsistent here not to.

Reply:
@nomasprime Again I have been consistent with the existing codebase. I think given time constraints we should look to refactor anything we feel strongly about after Private Beta. Additionly standard rake task has not flagged this up. Maybe rubocop would have, but we have inherited using standardrb for code linting

Reply:
I thought the same. Let's put a pin in it.